kolibri/core/assets/src/utils/i18n.js

```diff
@@ -132,13 +132,16 @@ class Translator {
    * @param {object} defaultMessages - an object mapping message ids to default messages.
    */
   constructor(nameSpace, defaultMessages) {
-    this.nameSpace = nameSpace;
-    this.defaultMessages = defaultMessages;
+    this._nameSpace = nameSpace;
+    this._defaultMessages = defaultMessages;
+    for (const key in defaultMessages) {
+      this[key] = this.$tr.bind(this, key);
```

Comment:
One thing I'd miss with this solution as it is now is that I do appreciate the `$tr` as a unique token to indicate visually the context that `$tr('kolibriLibrary')` is an i18n string whereas `kolibryLibrary()` as a function doesn't say anything about it's nature as an i18n message.

We have in the past discussed how we ought to be labeling our translation keys with language that gives it some more context. For example, if the `kolibriLibrary` function was instead maybe `kolibriLibraryTitle` - then we can have some common naming conventions a la `*Title, *Label, *Message, etc` that would give that context.

Another thought is that perhaps we could add some kind of token to the name of the key to indicate by its name that it is a i18n string. I'm thinking something like this maybe:

```
// change the line this comment is on to:
this["$tr_" + key] = this.$tr.bind(this, key);

// Then when you destructure it:
const { $tr_kolibriLibrary } = learnStrings;
```

Idk maybe that's horrible 😆 just a thought anyway.

Reply:
For those of us who learned BASIC, adding `$` to the end of the message name would be one way to do it!

```
this[key + '$'] = this.$tr.bind(this, key);
```

```
const { kolibriLibrary$ } = learnStrings;
```

Reply:
@nucleogenesis That's a good note. I'd too appreciate being able to recognize functions related to translation strings. When it comes to a particular format, I feel inclined towards `kolibriLibrary$` because it has the same clarity as `$tr_kolibriLibrary` while being simpler.

Reply:
Agreed! Appending a $ is perfect

Reply:
Have updated accordingly.